editor/js/Loader.js

```diff
@@ -768,7 +768,15 @@ function Loader( editor ) {
 
 			default:
 
-				console.error( 'Unsupported file format (' + extension + ').' );
+				alert( [
+
+					editor.strings.getKey( 'prompt/file/import/fail' ) + ': ' +
+					editor.strings.getKey( 'prompt/file/import/unsupportedFileFormat' ) + ' (' + extension + ')',
+					'',
+					editor.strings.getKey( 'prompt/file/import/supportedFileFormat' ) + ': ',
+					'3dm, 3ds, 3mf, amf, dae, drc, fbx, glb, gltf, js, json, kmz, ldr, mpd, md2, obj, pcd, ply, stl, svg, usdz, vtk, vtp, wrl, xyz, zip(fbx), zip(glb), zip(gltf), zip(obj)'
```

Comment:
I'm not a fan of such listings since they are hard to maintain. A simple message like "Unsupported file format" is sufficient, imo.

Reply:
The listing is just extracted from cases of this `switch` https://github.com/mrdoob/three.js/blob/79c34909e6af9250c38182665e3bd4896149f0ea/editor/js/Loader.js#L80

btw, is this supported formats list documented somewhere?

Reply:
> btw, is supported formats list documented somewhere?

No, and I don't think documenting is worth the effort. I understand it's desirable to have such things documented but according to my experience it's was always a pain to keep these lists up to date (e.g. like supported WebGL extensions, supported WebGL 2 features, the Web APIs used by the engine). Eventually we have always removed such lists because the got out-of-sync. Especially since enthusiastic contributors who added such listings in the first placed vanished and didn't care about them anymore. So it was the core team that had to endure the complains from user side (why is your stuff not properly documented?!?) and maintain all these things. After all, I prefer to have as less as support intensive things in the project as possible.

Reply:
I see your point, lets brainstorm a way helping both maintainers and users? e.g. slightly modify Loader.js to introspect the  switch cases and shown as hint.

Reply:
done.